toolchain/personae.md

```diff
@@ -0,0 +1,56 @@
+---
+title: The Good Docs Toolchain
+---
+
+This document summarizes the personae for The Good Docs Toolchain, a collection of tools designed to help users create better technical documentation.
```

Comment:
As per meeting discussion, it appears "personas" is more commonly used than "personae". Find and replace in rest of doc. Filename should change too.